### ISSUE TITLE
Tagged release for PHP7?

### ISSUE THREAD

**Issue by afk11** [sop/x509#2]
Hey, noticed the other packages were tagged with the PHP7 changes, but not this one. Is there anything that needs doing that can happen, or could you tag it when you get a chance?

**Comment by sop** [comment]
There was something that i thought should be done before releasing, but i had to jump on another task. I'll have a look and push a tag asap :).
